src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
 #Application Properties
 #server.port=3001
 #Conexion MongoDB
-#spring.data.mongodb.uri=
+spring.data.mongodb.uri=mongodb+srv://at-sso-api:4gil3th0ught@cluster0.vliih.mongodb.net/at-sso-db?retryWrites=true&w=majority
```

Comment:
Remember we must not send connections strings to the server. those are only for local testing. Please remove it

Reply:
done